app/widgets/common/secrets/SecretActions.ts

```diff
@@ -30,6 +30,10 @@ export default class SecretActions {
         return this.manager.doGet(`/secrets/${key}`);
     }
 
+    doGetSkipValue(key: Secret['key']): Promise<Secret> {
+        return this.manager.doGet(`/secrets/${key}?_skip_value=true`);
+    }
+
```

Comment:
1. I think the type is not fully accurate (as you don't have `value` inside).
2. I suggest also to change function name

```suggestion
    doGetSecretWithoutValue(key: Secret['key']) {
        return this.manager.doGet<Omit<Secret, 'value'>>(`/secrets/${key}?_skip_value=true`);
    }

```

Reply:
+1 on a different function name

`doGetSkipValue` seems related to its URL parameters but from another developer perspective, it may be confusing as use case of that parameter may not be known by the developer

Making the function/variable names related to their purpose, rather than their insides, seems to be a good strategy and a good rule of thumb 💪

Reply:
1. Changed name, you're absolutely right guys - that's way better than mine idea 😅 
2. I went for  `doGetWithoutValue(key: Secret['key']): Promise<Omit<Secret, 'value'>> {` because when I modified `return` - I got TS error

Reply:
Ad 2. 
```
    doGetWithoutValue(key: Secret['key']) {
        return this.manager.doGet<Omit<Secret, 'value'>>(`/secrets/${key}?_skip_value=true`);
    }
```
^ it doesn't work? 🤔 I just checked locally and have no issues. Am I missing something?

Reply:
okay, I'm pretty sure that when I was checking earlier it was throwing a TS error, however it's working now 🤔 I might have missed some type earlier, but since it's working right now I updated it accordingly: https://github.com/cloudify-cosmo/cloudify-stage/pull/2427/commits/10585cbba8c7cb6f4871c7caf838a6f657d3dde2 ✔️